README.md

```diff
@@ -150,6 +150,140 @@ Comparison:
 
 ```
 
+## Testing
+
+### Happy Path
+
+When testing positive cases of a Verbalize interactor, it is recommended to test using the `call!` class method and
+assert on the result.  This implicitly ensures a successful result, and your tests will fail with a bang! if 
+something goes wrong:
+
+```ruby
+class MyInteractor
+  include Verbalize
+  
+  input :a
+  
+  def call
+    fail!('`a` must be less than 100!') if a >= 100
+    a + 1
+  end
+end
+
+it 'returns the expected result' do
+  result = MyInteractor.call!(a: 50)
+  expect(result).to eq 51
+end
+```
+
+### Sad Path
+
+When testing negative cases of a Verbalize interactor, it is recommended to test using the `call` non-bang 
+class method.  Use of `call!` here is not advised as it will result in an exception being thrown. Set assertions 
+on both the outcome and error value of the result:
+
+```ruby
+class MyInteractor
+  include Verbalize
+  
+  input :a
+  
+  def call
+    fail!('${a} is not less than 100!') if a >= 100
```

Comment:
Still needs fixed. And actually, this is inconsistent with the previous example using `a`. Don't really care which as long as it's consistent.